libs/providers/flagd-web/src/lib/flagd-web-provider.ts

```diff
@@ -131,8 +133,9 @@ export class FlagdWebProvider implements Provider {
             }
           }
         },
-        () => {
-          this._logger?.error(`${FlagdWebProvider.name}: could not establish connection to flagd`);
+        (err) => {
+          this._logger?.error(`${FlagdWebProvider.name}: could not establish connection to flagd, ${err?.message}`);
+          this._logger?.debug(err?.stack);
```

Comment:
This is unrelated, but I realized we didn't have error context here so I added some.